src/styles/main.scss

```diff
@@ -1,3 +1,6 @@
-body {
-  margin: 0;
-}
+@import "./utils/reset.scss";
+@import "./utils/normalize.scss";
+@import "./utils/variables";
+@import "./utils/functions";
+
+@import "./blocks/calendar.scss";
```

Comment:
Importing 'calendar.scss' is not following BEM naming convention. Change the file name to '_calendar.scss' and fix the import statement.